macroslib/src/cpp/mod.rs

```diff
@@ -533,6 +537,23 @@ fn merge_rule(ctx: &mut CppContext, mut rule: TypeMapConvRuleInfo) -> Result<()>
     Ok(())
 }
 
+fn cache_f_code(cached_code: &mut Vec<(SmolStr, String)>, f_module_name: &SmolStr, f_code_str: &str) -> bool {
+    for (module_name, code) in cached_code.iter() {
+        if (module_name == f_module_name) && (code.as_str() == f_code_str) {
+            debug!("module code entry already cached.");
+
+            // Return TRUE as this part of code is already cached.
+            return true;
+        }
+    }
+
+    debug!("creating new code cache entry.");
+    cached_code.push((f_module_name.clone(), String::from(f_code_str)));
+
+    // Return FALSE as this part of code is just added to code cache.
```

Comment:
again comment about fact that is clear from previous and next lines

Reply:
I'll remove the comment.
Do I keep the `debug!()` or shall I remove it too ?